sky/services/editing/editing.mojom

```diff
@@ -0,0 +1,53 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module editing;
+
+enum TextAffinity {
+  upstream,
+  downstream,
+};
+
+struct TextSelection {
+  int32 base;
+  int32 extent;
+  TextAffinity affinity = TextAffinity.downstream;
+  bool is_directional;
+};
+
+enum SubmitAction {
+  DONE,
+};
+
+enum KeyboardType {
+  TEXT,
```

Comment:
The casing of the names of the enum values is inconsistent (with `TextAffinity`)